conda/environments/rapids_triton_dev.yml

```diff
@@ -5,5 +5,7 @@ channels:
 dependencies:
   - ccache
   - cmake>=3.21,<3.23
+  - libstdcxx-ng<=11.2.0
+  - libgcc-ng<=11.2.0
```

Comment:
Can you add a note in the PR description or in the code of how the libstdc++ and symbol looked like in the error, to assist people debugging future similar things